packages/leancode_hooks/lib/src/use_decarative_text_editing_controller.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+TextEditingController useDeclarativeTextEditingController({
+  required String text,
+}) {
+  final controller = useTextEditingController(text: text);
+
+  useEffect(
+    () {
+      if (controller.text != text) {
+        WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
+          controller.text = text;
+        });
+      }
+      return null;
+    },
+    [text],
+  );
+
+  return controller;
+}
```

Comment:
@shilangyu Maybe we could merge this somehow with mine `useSyncedTextEditingController`?

I thought of using `useDeclarativeTextEditingController` inside `useSyncedTextEditingController`.e